tools/.github-cla-signers

```diff
@@ -4,3 +4,4 @@ lucasmoura
 nishigori
 tomponline
 TheRealFalcon
+bipinbachhao
```

Comment:
Let's keep this file alphabetised, please!

Reply:
pushed the new commit